src/Base/AbstractResource.php

```diff
@@ -197,6 +197,32 @@ public function destroy(Request $request, $id)
         return back()->withSuccess('Resource successfully deleted!');
     }
 
+    /**
+     * @param Request $request
+     * @param $id
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function toggle(Request $request, $id)
+    {
+        $column = $request->get('column', null);
+
+        abort_unless($column, 400);
+
+        $model = $this->model;
```

Comment:
Te var izmantot `$this->getModel()`

Reply:
Kāpēc izsaukt funkciju, ja man ir tieša pieeja propertijam?

Reply:
1. Vienots pieraksts, šim pierakstam cenšamies visās metodēs pieturēties
2. Ja nākotnē mainīsim, kā dabūjam "model", tad ar getModel() problēmu nebūs